src/App.tsx

```diff
@@ -16,34 +17,132 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [sortedGoods, setSortedGoods] = useState([...goodsFromServer]);
+  const [reversePressed, setReversePressed] = useState(false);
+  const [alphabeticallyPressed, setAlphabetically] = useState(false);
+  const [byLengthPressed, setByLength] = useState(false);
+
+  enum SortType {
+    Alphabetically = 'alphabetically',
+    ByLength = 'byLength',
+    Reverse = 'reverse',
+    Reset = 'reset',
+  }
+
+  const sortGoods = (sortType: SortType) => {
+    let sorted = [...sortedGoods];
+
+    switch (sortType) {
+      case 'alphabetically':
```

Comment:
The `case` statement should use `SortType.Alphabetically` instead of the string 'alphabetically'. This ensures consistency with the `enum` usage.